docs/whats-new/preview-4.md

```diff
@@ -578,8 +578,80 @@ This release we've enabled a Visual Studio publish experience for .NET Aspire ap
 
 ## Azure Developer CLI
 
-The Azure Developer CLI (azd) has been updated to support the new manifest types introduced in preview 4. Azd supports prompting for parameter resources, and supports the bicep resource which enables deploying any thing can be described using a bicep module.
+The [Azure Developer CLI](https://aka.ms/azd) (azd) has been updated to support the new manifest types introduced in preview 4. Azd supports prompting for parameter resources, and supports the bicep resource which enables deploying any thing. All Azure resources can be described using a bicep module.
 
 In addition to new features, azd will automatically create secrets in Azure Container Apps for any parameter marked as a secret, and any environment variable that references a secret or uses a connection string.
 
-TODO: Expand with images
+### Prompting for parameters
+
+When azd detects a parameter, it prompts for a value. For example, consider the following app host code:
```

Comment:
```suggestion
When azd detects a parameter.v0 resource in the manifest, it will prompt for a value. For example, consider the following code in the apphost project:
```